generator.rb

```diff
@@ -0,0 +1,42 @@
+def run_generator
+  puts "What is the first word to combine?"
+  word_A = gets.chomp
+
+  until (word_A.length > 1 && word_A =~ /[aeiou]/)
```

Comment:
Suggestion:  Do you notice how these two loops look identical?  You could create a method and reuse the code.  That might condense things a bit.